connexion/http_facts.py

```diff
@@ -11,5 +11,5 @@
     "options",
     "head",
     "patch",
-    "trace"
+#    "trace"  #  FIXME: rpolli
```

Comment:
@kigawas Sanic does not support `trace`. Ideas welcome!

Reply:
Trace method is rather rare, do we really need it?

Reply:
It's in connexion code, so we should ask them. @dtkav